### PR TITLE
[CS] add fluent method coding standard

### PR DESCRIPTION
**Pull request by TomasVotruba**
Waits on https://github.com/eonx-com/easy-monorepo/pull/384

Chain method calls have 1 line = 1 call standard. It makes it easier to see what each line does. Also, will be required for formatting code that Rector prints.

## Also

- Prepare empty Rector CI

### CODE REVIEW

File: packages/EasyErrorHandler/composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     },
     "require-dev": {
         "phpunit/phpunit": "^7.5 || ^8.4",
-        "laravel/lumen-framework": "^5.5 || ^6.0",
+        "laravel/lumen-framework": "^5.5",
```
Comment (TomasVotruba) [inline packages/EasyErrorHandler/composer.json:13]: This makes it compatible with `composer.json` in the root

File: .editorconfig [hunk 2]
```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+indent_style = space
+indent_size = 4
```
Comment (TomasVotruba) [inline .editorconfig:9]: This one standardizes spaces to single way in every file

File: composer.json [hunk 3]
```diff
@@ -158,23 +159,23 @@
             "@check-security",
             "@check-tests"
         ],
-        "check-ecs": "vendor/bin/ecs check --clear-cache",
-        "check-monorepo": [
-            "@composer validate --strict"
-        ],
-        "check-phpstan": "vendor/bin/phpstan analyse --error-format symplify",
+        "check-ecs": "vendor/bin/ecs check --ansi",
+        "check-monorepo": "vendor/bin/monorepo-builder validate --ansi",
```
Comment (TomasVotruba) [inline composer.json:163]: This return monorepo root <=> packages' `composer.json` version validation

**Comment by TomasVotruba** [comment]
Ready to merge :heavy_check_mark:

**Review by natepage — APPROVED** [review]

**Pull request merged**